core/domain/image_services_test.py

```diff
@@ -100,15 +103,11 @@ def is_different(file1, file2):
             Returns:
                 bool. Returns whether the images are different.
             """
-            from PIL import ImageChops
             image1 = Image.open(io.BytesIO(file1)).convert('RGB')
             image2 = Image.open(io.BytesIO(file2)).convert('RGB')
             diff = ImageChops.difference(image1, image2)
 
-            if diff.getbbox():
-                return False
-            else:
-                return True
+            return not diff.getbbox()
```

Comment:
Is this flipped from what it should be (given the new definition of the function)?

Reply:
I actually messed up in the first implementation. Also I originally added a comment but for some reason the function wasn't working so I removed it for testing and forgot to add it back in. I'll add it back in with the next commit to make it clearer.

Gist is bounding box finds non zero elements which means diff.getbbox() is empty, then the two images are identical